src/lib/slider/_slider-theme.scss

```diff
@@ -5,7 +5,6 @@
 @mixin md-slider-theme($theme) {
   $accent: map-get($theme, accent);
 
-  // TODO(iveysaur): Find an implementation to hide the track under a disabled thumb.
```

Comment:
Isn't that already implemented? (I probably miss something)

![](https://i.gyazo.com/5f280846cd97e851579f623903352d6d.png)